CHANGES.rst

```diff
@@ -4,7 +4,8 @@ CHANGES
 1.4 (unreleased)
 ================
 
-- Nothing changed yet.
+- Check for builtins (Python 3) everywhere that we check for __builtin__
+  (Python 2).
```

Comment:
I think `__builtin__` needs some ReST escaping.

    ``__builtin__``

is probably the best option (I hate how `\_\_builtin\_\_` looks).

Reply:
Done, thanks.